File: app/src/main/java/com/kota/Telnet/TelnetClientListener.java
package com.kota.Telnet;

public interface TelnetClientListener {
    void onTelnetClientConnectionClosed(TelnetClient telnetClient);

    void onTelnetClientConnectionFail(TelnetClient telnetClient);

    void onTelnetClientConnectionStart(TelnetClient telnetClient);

    void onTelnetClientConnectionSuccess(TelnetClient telnetClient);
}

File: app/src/main/java/com/kota/telnet/TelnetClientListener.kt
package com.kota.telnet

interface TelnetClientListener {
    fun onTelnetClientConnectionClosed(telnetClient: TelnetClient?)

    fun onTelnetClientConnectionFail(telnetClient: TelnetClient?)

    fun onTelnetClientConnectionStart(telnetClient: TelnetClient?)

    fun onTelnetClientConnectionSuccess(telnetClient: TelnetClient?)
}
